src/pages/api/simulation.ts

```diff
@@ -0,0 +1,28 @@
+import { TENDERLY_ACCESS_KEY, TENDERLY_PROJECT, TENDERLY_USER } from "../../consts/config";
+
+
+export default async function handler(req,res){
+  const data=req.body
+   
+  const resp = await fetch(
+    `https://api.tenderly.co/api/v1/account/${TENDERLY_USER}/project/${TENDERLY_PROJECT}/simulate`,
```

Comment:
Check that these env vars are defined and throw a helpful here at the top if they're not